backend/dataall/searchproxy/indexers.py

```diff
@@ -1,334 +1,75 @@
 import logging
 
 from sqlalchemy import and_
-from sqlalchemy.orm import with_expression
 
-from .upsert import upsert
 from .. import db
 from ..db import models
+from dataall.searchproxy.upsert import BaseIndexer
 
 log = logging.getLogger(__name__)
 
 
-def get_target_glossary_terms(session, targetUri):
-    q = (
-        session.query(models.TermLink)
-        .options(
-            with_expression(models.TermLink.path, models.GlossaryNode.path),
-            with_expression(models.TermLink.label, models.GlossaryNode.label),
-            with_expression(models.TermLink.readme, models.GlossaryNode.readme),
-        )
-        .join(
-            models.GlossaryNode, models.GlossaryNode.nodeUri == models.TermLink.nodeUri
-        )
-        .filter(
-            and_(
-                models.TermLink.targetUri == targetUri,
-                models.TermLink.approvedBySteward.is_(True),
+class DashboardIndexer(BaseIndexer):
```

Comment:
I believe this would be moved to the Dashboard module in the future when it gets moved. Maybe leave here a TODO comment?

Reply:
Sure!